### PR TITLE
cleaned locations

### PR DESCRIPTION
**Pull request by domdinicola**
removed old command
flake8
isort

### CODE REVIEW

File: EquiTrack/locations/tasks.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from carto.sql import SQLClient
 
 from EquiTrack.celery import app
+
 from .models import Location
```
Comment (reinbach) [inline EquiTrack/locations/tasks.py]: Change to absolute import

File: EquiTrack/locations/tests/test_models.py [hunk 2]
```diff
@@ -1,16 +1,9 @@
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-from __future__ import unicode_literals
+from __future__ import absolute_import, division, print_function, unicode_literals
```
Comment (reinbach) [inline EquiTrack/locations/tests/test_models.py]: I believe the preference is to have these on separate lines

Comment (domdinicola) [reply EquiTrack/locations/tests/test_models.py]: Do you think would be useful to handle 'Multi line output modes' with 'Vertical Hanging Indent' mode (option 3)? (check https://pypi.python.org/pypi/isort)

Comment (reinbach) [reply EquiTrack/locations/tests/test_models.py]: I guess there is no mandated way that these are imported, I've always seen them on separate lines and I've alwats done that as well.

Comment (domdinicola) [reply EquiTrack/locations/tests/test_models.py]: To me is fine, I've been cleaning those in all the code :/

Comment (caktus-philip) [reply EquiTrack/locations/tests/test_models.py]: I agree with Greg. Obviously both are valid syntax and I don't have a strong personal preference for either version. However, I think it's good practice to respect the established style on a project even if that style is not my personal preference.

Comment (domdinicola) [reply EquiTrack/locations/tests/test_models.py]: done!

**Pull request merged**